crates/ruma-client-api/src/media.rs

```diff
@@ -21,3 +21,39 @@ fn default_download_timeout() -> Duration {
 fn is_default_download_timeout(timeout: &Duration) -> bool {
     timeout.as_secs() == 20
 }
+
+/// Checks whether a given `Content-Type` is considered "safe" for having a `Content-Disposition` of
+/// `inline`. This should be used on `/download` only, as `/thumbnail` should only ever return safe
+/// types as thumbnails are generated by the server.
+pub fn is_safe_content_type(content_type: &str) -> bool {
```

Comment:
Maybe we should change the name to `is_safe_inline_content_type`, so it's clearer what it is safe for.